Automattic/newspack-content-converter
# Patcher for converting Largo module shortcodes to pullquotes

Closes #9 

This PR adds a patcher that converts shortcode blocks containing the Largo theme's `module` shortcode into Pullquote blocks. It should match up to the original settings pretty well when possible.

To test:
1. Run unit tests. Verify they pass.
2. Add a shortcode block to a post and add any of the following to it:
```
[module align="left" width="half" type="pull-quote"]"The Board will add this to the ongoing investigation."
—New Beginnings attorney Michelle Craig[/module]

[module align="right” width=”half” type=”aside”]
<h6 class="p1">Previous coverage: <a href="https://thelensnola.org/2019/03/22/f-to-d-grade-changes-at-kennedy-high-school-are-suspicious-former-administrator-says/">F to D grade changes at Kennedy High School are suspicious, former administrator says</a></h6>
[/module]

[module align=”left” width=”full” type=”pull-quote”]”The first step is to speak about the issue more honestly, to understand the ways in which the climate crisis touches every aspect of our humanity.”
—Nathaniel Rich[/module]
```
3. Run the patcher and verify the shortcode blocks containing `module` shortcodes are converted into valid pullquote blocks.
